feature_server/commands.py

```diff
@@ -979,42 +1014,30 @@ def handle_command(connection, command, parameters):
         command_func = commands[command]
     except KeyError:
         return # 'Invalid command'
+    aspec = command_func.argspec
```

Comment:
Can we rename the local variable here from `aspec` instead of `argspec`? I'm almost certain it was a typo.